File: Entidades/ProductoEN.vb

Public Class ProductoEN
    Private _codprod As Integer
    Public Property CodProd() As Integer
        Get
            Return _codprod
        End Get
        Set(ByVal value As Integer)
            _codprod = value
        End Set
    End Property

    Private _codhistorico As Integer
    Public Property CodHistorico() As Integer
        Get
            Return _codhistorico
        End Get
        Set(ByVal value As Integer)
            _codhistorico = value
        End Set
    End Property


    Private _nombre As String
    Public Property Nombre() As String
        Get
            Return _nombre
        End Get
        Set(ByVal value As String)
            _nombre = value
        End Set
    End Property

    Private _descripcion As String
    Public Property Descripcion() As String
        Get
            Return _descripcion
        End Get
        Set(ByVal value As String)
            _descripcion = value
        End Set
    End Property

    Private _cantidad As Integer
    Public Property Cantidad() As Integer
        Get
            Return _cantidad
        End Get
        Set(ByVal value As Integer)
            _cantidad = value
        End Set
    End Property

    Private _precio As String
    Public Property Precio() As String
        Get
            Return _precio
        End Get
        Set(ByVal value As String)
            _precio = value
        End Set
    End Property


    Private _sector As String
    Public Property Sector() As String
        Get
            Return _sector
        End Get
        Set(ByVal value As String)
            _sector = value
        End Set
    End Property

    Private _activo As Boolean
    Public Property Activo() As Boolean
        Get
            Return _activo
        End Get
        Set(ByVal value As Boolean)
            _activo = value
        End Set
    End Property

    Private _dvh As Integer
    Public Property DVH() As Integer
        Get
            Return _dvh
        End Get
        Set(ByVal value As Integer)
            _dvh = value
        End Set
    End Property
End Class

File: Entidades/ProductoEN.cs

namespace Entidades
{
    public class ProductoEN
    {
        private int _codprod;

        public int CodProd
        {
            get
            {
                return _codprod;
            }

            set
            {
                _codprod = value;
            }
        }

        private int _codhistorico;

        public int CodHistorico
        {
            get
            {
                return _codhistorico;
            }

            set
            {
                _codhistorico = value;
            }
        }

        private string _nombre;

        public string Nombre
        {
            get
            {
                return _nombre;
            }

            set
            {
                _nombre = value;
            }
        }

        private string _descripcion;

        public string Descripcion
        {
            get
            {
                return _descripcion;
            }

            set
            {
                _descripcion = value;
            }
        }

        private int _cantidad;

        public int Cantidad
        {
            get
            {
                return _cantidad;
            }

            set
            {
                _cantidad = value;
            }
        }

        private string _precio;

        public string Precio
        {
            get
            {
                return _precio;
            }

            set
            {
                _precio = value;
            }
        }

        private string _sector;

        public string Sector
        {
            get
            {
                return _sector;
            }

            set
            {
                _sector = value;
            }
        }

        private bool _activo;

        public bool Activo
        {
            get
            {
                return _activo;
            }

            set
            {
                _activo = value;
            }
        }

        private int _dvh;

        public int DVH
        {
            get
            {
                return _dvh;
            }

            set
            {
                _dvh = value;
            }
        }
    }
}